kernel/kernel-api/src/main/java/io/delta/kernel/internal/replay/PaginatedScanFilesIteratorImpl.java

```diff
@@ -57,11 +57,28 @@ public class PaginatedScanFilesIteratorImpl implements PaginatedScanFilesIterato
   private long numScanFilesReturned;
 
   /**
-   * The name of the last log file that was read. This value corresponds to the one saved in the
-   * page token if present.
+   * The name of the last log file that was read during pagination.
+   *
+   * <p>This value is used to track which log file the current scan is processing.
+   *
+   * <p>Initialization: - If the pagination token includes a log file path, this value is
+   * initialized from it. - If the pagination token does not include a log file path (i.e., the
```

Comment:
I see you are using `-` for different points. As you can see, the java formatter doesn't like this. Can you please use `<ul><li>...`